Makefile.conf

```diff
@@ -35,9 +35,9 @@ COQMF_CMDLINE_COQLIBS =
 ###############################################################################
 
 COQMF_LOCAL=0
-COQMF_COQLIB=/usr/local/lib/coq/
-COQMF_DOCDIR=/usr/local/Cellar/coq/8.11.1/share/doc/coq/
-COQMF_OCAMLFIND=/usr/local/opt/ocaml-findlib/bin/ocamlfind
+COQMF_COQLIB=/Applications/CoqIDE_8.11.1.app/Contents/Resources/lib/coq/
```

Comment:
This file obviously needs to be ignored

Reply:
Lets make a follow pull request for that.